internal/osv/osv.go

```diff
@@ -156,7 +157,16 @@ func MakeRequest(request BatchedQuery) (*BatchedResponse, error) {
 
 // Get a Vulnerabiltiy for the given ID.
 func Get(id string) (*models.Vulnerability, error) {
-	resp, err := http.Get(GetEndpoint + "/" + id)
+	var resp *http.Response
```

Comment:
Can you move this logic into a separate function, that way it can be reused for the initial query above as well?

Reply:
Done!